internal/v1/handler_blueprints_test.go

```diff
@@ -1696,3 +1696,238 @@ func TestFixupBlueprint(t *testing.T) {
 		require.NoError(t, srv.DB.DeleteBlueprint(context.Background(), bpID, "000000", "000000"))
 	}
 }
+
+func TestBlueprintComplianceSnapshot(t *testing.T) {
+	if runtime.GOOS == "darwin" {
+		t.Skip("crypt() not supported on darwin")
+	}
+
+	ctx := context.Background()
+	dbase, srvURL, shutdownFn := makeTestServer(t, nil)
+	defer shutdownFn(t)
+
+	body := map[string]interface{}{
+		"name":        "BlueprintWithCompliance",
+		"description": "Blueprint with compliance policy",
+		"customizations": map[string]interface{}{
```

Comment:
Let's keep the blueprint minimal here, with only the openscap customization set.